sync/game_sync_classes.py

```diff
@@ -0,0 +1,13 @@
+from sync.games import *
+
+# Add games here as they are supported.
+gameSyncMap = {
+    'POKEMON BLUE': pkmnrb.PokemonRedBlueSync,
+    'POKEMON RED': pkmnrb.PokemonRedBlueSync
+}
+
+def get_game_sync_class(gameName):
+    game_sync = gameSyncMap.get(gameName)
+    if game_sync == None:
```

Comment:
this if block is redundant, if game sync is None it gets returned as such anyway